recipe/meta.yaml

```diff
@@ -1,33 +1,33 @@
 {% set name = "thinc" %}
-{% set version = "8.1.10" %}
+{% set version = "8.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6c4a48d7da07e044e84a68cbb9b22f32f8490995a2bab0bfc60e412d14afb991
+  sha256: 6e85b944672c0f95241a71f67f9882e1ab319c449a47740b0d159f4cf86d1587
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   skip: True  # [py<36]
-  # 2021/11/11: skip s390x as many dependencies are not on this platform: 
-  # e.x., preshed, cython-blis, cymem, murmurhash. 
+  # 2021/11/11: skip s390x as many dependencies are not on this platform:
+  # e.x., preshed, cython-blis, cymem, murmurhash.
   skip: True  # [s390x]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - cymem 2.0.6
-    - cython 0.29.32
-    - cython-blis 0.7.9
-    - murmurhash 1.0.7
+    - cymem
+    - cython 0.29
+    - cython-blis
+    - murmurhash
```

Comment:
You need to use the same constraint as in the run section. Same for all the other python host deps. These modules are imported in cython files and I think that we need to respect the same constraints in build and runtime or at least follow https://github.com/explosion/thinc/blob/b183006daa740e648cca997e0cca29239e18e575/pyproject.toml#L4-L8.

See https://github.com/explosion/thinc/blob/b183006daa740e648cca997e0cca29239e18e575/thinc/backends/numpy_ops.pyx#L16C30-L16C36.

Reply:
I agree, I addressed your comments.